Library/Homebrew/tap.rb

```diff
@@ -229,7 +229,7 @@ def install(options = {})
       raise
     end
 
-    link_manpages
```

Comment:
I'd rather we kept this into two separate function calls. Apologies if that contradicts other maintainers.